core/src/main/java/de/jplag/reporting/jsonfactory/ComparisonReportWriter.java

```diff
@@ -104,7 +104,8 @@ private Match convertMatchToReportMatch(JPlagComparison comparison, de.jplag.Mat
     }
 
     private String relativizedFilePath(File file, Submission submission) {
-        return submission.getRoot().toPath().relativize(file.toPath()).toString();
+        // add root directory into JSON file#710
```

Comment:
Please don't write comments that reference the issue or comments that explain *what* a code line does. If comments are used, they should only described *why* something happens

Reply:
Thanks for your hints, i will pay attention to this.